ballerina/modules/soap12/tests/http_soap_service.bal

```diff
@@ -63,7 +63,15 @@ service / on new http:Listener(9090) {
             response.setPayload(payload);
             return response;
         }
-        return error("Invalid action is found");

```

Comment:
Shall we add a test case for this scenario as well? Returning an error from the service, which results in a 500 status code from the server side. Just to check how it is handled by our soap client.

Reply:
Added in 
30e4376